lib/normalizer.c

```diff
@@ -3488,6 +3488,26 @@ grn_nfkc_normalize_unify_iteration_mark(grn_ctx *ctx,
     return unified_buffer;
   }
 #  undef N_KATAKANA_BYTES
+  else if (current_length == 3 && current[0] == 0xe3 && current[1] == 0x80 &&
+           current[2] == 0xbb && previous_length >= 3 &&
+           GRN_CHAR_TYPE(grn_nfkc_char_type(previous)) == GRN_CHAR_KANJI) {
+    /**
+     * U+303B VERTICAL IDEOGRAPHIC ITERATION MARK
+     * For kanji iteration mark, we simply repeat the previous kanji character.
+     * This implementation only handles simple cases as follows.
+     * - U+5404 U+303B -> U+5404 U+5404
```

Comment:
Could you add character name to all "U+XXXX" like we did in the above line ("* U+303B VERTICAL IDEOGRAPHIC ITERATION MARK")?

```suggestion
     * - U+5404 U+303B -> U+5404 U+5404
     *   ("CJK UNIFIED IDEOGRAPH-5404" "VERTICAL IDEOGRAPHIC ITERATION MARK" ->
     *    "CJK UNIFIED IDEOGRAPH-5404" "CJK UNIFIED IDEOGRAPH-5404")
```

Reply:
fix: a2e1e9b50b66d05e508df21b7feb4736398b8bc8 Sure.